script/testing/jdbc/src/PelotonTest.java

```diff
@@ -14,7 +14,7 @@
 import org.postgresql.util.*;
 
 public class PelotonTest {
-  private final String url = "jdbc:postgresql://localhost:54321/";
+  private final String url = "jdbc:postgresql://localhost:5432/";
```

Comment:
@eric-haibin-lin Also, could you add the ../script/testing/jdbc/stats_log file to gitignore?